salad-feign/src/main/java/org/techsalad/feign/controller/SaladController.java

```diff
@@ -0,0 +1,26 @@
+package org.techsalad.feign.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.techsalad.feign.client.SaladClient;
+
+@RestController
+@RequestMapping(path = "/feign")
+@AllArgsConstructor
+public class SaladController {
+    private final SaladClient saladClient;
```

Comment:
I might misunderstood, but from the sources it seems that:  `SaladController` calls `FeigClient`, that calls the rest endpoints of the `SaladController`, so it seems to run the endless loop. Feel free to correct me, if I misunderstood.

Reply:
ok, clarified via slack, not an issue, marking resolved